Reuse image on Dockerhub

krumeich opened the pull request:
No need to build the entire image unless the underlying Perl version or the libs require updates

krumeich committed:
Reuse image on Dockerhub

No need to build the entire image unless the underlying Perl version or the libs require updates

sbrass reviewed (commented):
Does it make sense to reuse the branch variable for tagging the image? 

IMO, I'd like to introduce an env variable DOCKER_TAG which is set to BRANCH when empty. We have then flexibility and in case a single source of truth, however, at the cost of another variable.

krumeich commented:
Good idea! I'll look into this. Could take a few days, though. Holidays and stuff.

krumeich committed:
Introduced new env variable DOCKER_TAG

DOCKER_TAG sets the version of the Docker image used for building biber. See README.

krumeich commented:
Settled for the easy and obvious solution.

sbrass reviewed (approved):
Thanks for the good idea :)! Happy eastern...
